internal/proto/grpc_service_config/example_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//go:generate ./regenerate.sh
+
+package grpc_service_config_test
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/jsonpb"
+	wrapperspb "github.com/golang/protobuf/ptypes/wrappers"
+	scpb "google.golang.org/grpc/internal/proto/grpc_service_config"
+)
+
+// TestExampleMarshalToJSON is an example to print json format of xds_config.
```

Comment:
Name doesn't match.  I thought this was going to be an example without verification?

Reply:
Name fixed.

Examples don't output anything. Tests can print the json (to be copied or modified).